frontend/src/components/Forms/AddMinespaceUser.js

```diff
@@ -46,7 +46,7 @@ export const AddMinespaceUser = (props) => (
               component={renderConfig.MULTI_SELECT}
               data={props.mines}
               onChange={props.handleChange}
-              input={{ onBlur: props.handleChange, onSearch: props.handleSearch }}
+              onSearch={props.handleSearch}
```

Comment:
When I first saw the above, I was curious as to how it was working, but didn't ask about it.... should have. 

This new implementation looks much better!